packages/contracts/deploy/new/10_framework/43_plugin_setup_processor_upgradeable_conclude.ts

```diff
@@ -0,0 +1,17 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {skipIfNotZkSync} from '../../helpers';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log(`Concluding Plugin Setup Processor deployment.\n`);
+
+  const {deployments} = hre;
+  hre.aragonToVerifyContracts.push(
+    await deployments.get('PluginSetupProcessor')
```

Comment:
```suggestion
    await deployments.get('PluginSetupProcessorUpgradeable')
```

is there a reason why you didn't put the `Upgradeable` name?

Reply:
https://github.com/aragon/osx/pull/596/commits/9cd35fc54b54e8cc8a4e89b8d5770435c80b6ffa